README.md

```diff
@@ -1 +1,142 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Scholl Library <a name="about-project"></a>
+
+This is Application that is designed to record what books are in the library and who borrows them.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+> This project requires you to have ruby on your Local Environment.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/manzitresor/school_library.git
+  cd school_library
+```
+
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage
+> To run the project, execute the following command:
+```sh
+   ruby person.rb
+```
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Manzi Tresor**
+
+- GitHub: [@githubhandle](https://github.com/manzitresor)
+- Twitter: [@githubhandle](https://twitter.com/MANZITresor3)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/manzi-tresor-783b4022a/)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add Interface
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the <a href="https://github.com/manzitresor/vet_clinic/issues">issues page</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project,
+- You can follow me on Github <a href ="https://github.com/manzitresor">[@githubhandle]</a> 
+- You can give me a star ⭐ to this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse community.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] MIT License link is broken, kindly endeavor to fix it, to make ReadMe more professional!

**Broken ReadMe license link:**
![image](https://github.com/manzitresor/school_library/assets/12745474/2f80143f-e259-4e8e-ae91-43a417c0f440)